pkg/artifactory/resource_artifactory_remote_repository.go

```diff
@@ -239,6 +239,30 @@ func resourceArtifactoryRemoteRepository() *schema.Resource {
 				Optional: true,
 				Default:  "",
 			},
+			"nuget": {
+				Type:     schema.TypeSet,
```

Comment:
Can we make this typeList. Set requires hashing functions. It shouldn't change what the schema looks like as HCL config but as it stands this may have issues with diffs.

Reply:
Yep, I'll update in the morning